<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>匹配本地影片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_path_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选取待匹配路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="movie_path_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选取电影存放路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="capture_path_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="movie_path_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="log_area">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>331</width>
     <height>152</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
